site-service/src/main/java/uk/ac/ox/ndph/mts/site_service/model/Attribute.java

```diff
@@ -48,6 +50,10 @@ public static Attribute fromString(String input) throws InitialisationError {
             return NAME;
         } else if (AttributeNames.ALIAS.nameof().equals(input)) {
             return ALIAS;
+        } else if (AttributeNames.PARENT_SITE_ID.nameof().equals(input)) {
```

Comment:
I don't like this else-if tower, can we just use some method on the AttributeNames enum instead?